nntrainer/models/neuralnet.cpp

```diff
@@ -838,7 +839,12 @@ int NeuralNetwork::train_run(std::function<bool(void *userdata)> stop_cb,
 
   auto train_epoch_end = [this, stop_cb, user_data](RunStats &stat,
                                                     DataBuffer &buffer) {
-    stat.loss /= static_cast<float>(stat.num_iterations);
+    try {
+      stat.loss /= static_cast<float>(stat.num_iterations);
```

Comment:
There is no way to be zero but we just need check if (stat.num_iterations == 0)

Reply:
[FIX] Thank you for Review